upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template

```diff
@@ -440,7 +425,7 @@ spec:
       hostNetwork: true
       containers:
         - name: node-driver-registrar
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.1.0
```

Comment:
registry.k8s.io supersedes k8s.gcr.io, so we probably should not change this.

Reply:
ok I was just copying values from openstack manifests.. I can change these back and verify that images exists